active/mergers.md

```diff
@@ -0,0 +1,51 @@
+# Mergers Team
+
+## Scope of responsibilities
+
+Mergers are a small set of people who merge pull requests to the [Django git repository](https://github.com/django/django/).
+
+Mergers will:
+- Merge pull requests when they can have determined that:
+  - There is consensus
+  - A DEP is approved or not needed
+  - The PR meets the documented code quality guidelines
+- Merge pull requests when the [Security Team](./security.md) directs something to be merged
+- Share knowledge and propose documentation changes about undocumented code quality or consistency practices
+- Request the Steering Council to vote on a decision that fails to reach consensus if a decision is needed
```

Comment:
- Perform backports of changes merged into the `main` branch to the stable branches following the [backporting policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions).

Reply:
```suggestion
- Request the Steering Council to vote on a decision that fails to reach consensus if a decision is needed
- Perform backports of changes merged into the `main` branch to the stable branches following the [backporting policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions)
```